app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  
+  validates :name,        presence: true
+  validates :email,       presence: true, format: { with:VALID_EMAIL_REGEX }
+  validates :password,    presence: true, length: { maximum: 10, minimum: 6 }
```

Comment:
I did not get any sort of validation warning / message from the password field. As a user I didn't know it had to be between 6 and 10.

Reply:
sorted

Reply:
Cool. I see this on page now.